zeus/monitor/energy.py

```diff
@@ -179,6 +180,7 @@ def __init__(
                 instantaneous power consumption with the window's execution time. This should
                 be a better estimate than zero, but it's still an approximation.
             log_file: Path to the log CSV file. If `None`, logging will be disabled.
+            backend: The backend framework. Defaults to `torch`
```

Comment:
```suggestion
            backend: Deep learning framework to use to synchronize GPU computations.
                Defaults to `"torch"`, in which case `torch.cuda.synchronize` will be used.
```
Just made it a bit more detailed. Same suggestion applies to `cuda_sync`.